src/pages/home/report/ReportActionItemMessage.js

```diff
@@ -42,4 +54,4 @@ ReportActionItemMessage.propTypes = propTypes;
 ReportActionItemMessage.defaultProps = defaultProps;
 ReportActionItemMessage.displayName = 'ReportActionItemMessage';
 
-export default withNetwork()(ReportActionItemMessage);
+export default compose(withNetwork(), withLocalize)(ReportActionItemMessage);
```

Comment:
NAB: May be wrap this to the next lines like the other components?

Reply:
Done